buildconf.py

```diff
@@ -83,7 +83,8 @@ def checkBaseName(package, info):
         if "$" in info["gitPackage"] and "*" not in package:
             info["basename"] = info["gitPackage"]
             info["gitPackage"] = info["gitPackage"].replace("$PACKAGE_BASENAME",
-                                                            package.split("/")[-1])
+                                                            '-'.join((package.split("/")[1:])))                                                           
```

Comment:
I think there are too many brackets.

Reply:
do you mean readability of the line?

Reply:
`(package.split("/")[1:])` is not necessary.